Assets/HoloToolkit/Input/Scripts/KeywordManager.cs

```diff
@@ -2,79 +2,97 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System.Collections.Generic;
-using System.Linq;
 using UnityEngine;
-using UnityEngine.Events;
 using UnityEngine.Windows.Speech;
 
 namespace HoloToolkit.Unity
 {
     /// <summary>
-    /// KeywordManager allows you to specify keywords and methods in the Unity
+    /// KeywordManager allows you to specify keywords in the Unity
```

Comment:
This removes the ability to hook keyword functions from the editor.
@keveleigh thoughts?